feat: add beagle flutter documentation (#717)
ZupIT/beagle-docs: content/en/resources/customization/beagle-for-flutter/customization/overview.md

---
title: Overview
weight: 1
type: overview
description: >-

>   You will find here the information on how to customize components, actions and operations in Beagle Flutter.

Reviewer: ```suggestion
 In this section, you will find information on how to customize components, actions, and operations in Beagle Flutter.
```
